config/shapes/variations.py

```diff
@@ -445,6 +474,20 @@ def __str__(self) -> str:
     ReplaceVariable("CMS_res_j_EraDown", "jerUncDown"),
 ]
 
+jet_es_hem = [
+    ReplaceVariable("CMS_scale_j_HEMIssue_EraUp", "jesUncHEMIssueUp"),
+    ReplaceVariable("CMS_scale_j_HEMIssue_EraDown", "jesUncHEMIssueDown"),
+]
+
+LHE_scale_norm_muR = [
+    AddWeight("LHE_scale_muR_normUp", Weight("(lhe_scale_weight__LHEScaleMuRWeightUp)", "muR2p0_muF2p0_weight")),
+    AddWeight("LHE_scale_muR_normDown", Weight("(lhe_scale_weight__LHEScaleMuRWeightDown)", "muR0p5_muF0p5_weight"))
+]
+
+LHE_scale_norm_muF = [
+    AddWeight("LHE_scale_muF_normUp", Weight("(lhe_scale_weight__LHEScaleMuFWeightUp)", "muR2p0_muF2p0_weight")),
+    AddWeight("LHE_scale_muF_normDown", Weight("(lhe_scale_weight__LHEScaleMuFWeightDown)", "muR0p5_muF0p5_weight"))
+]
```

Comment:
To make sure, the correct way would be to only vary muR or muF by 2 / 0.5 and keep the other one at 1.